pkgs/tools/security/iaito/default.nix

```
@@ -9,60 +9,77 @@
 , qttools
 , radare2
 , wrapQtAppsHook
-, nix-update-script
 }:
 
-# TODO MacOS support.
-# TODO Build and install translations.
-
 stdenv.mkDerivation rec {
   pname = "iaito";
-  version = "5.7.6";
-
-  src = fetchFromGitHub {
-    owner = "radareorg";
-    repo = pname;
-    rev = version;
-    sha256 = "sha256-PnIOoWPYLK30lMmLVctihCs7GBo0rTN8yetWAr21h9w=";
-  };
+  version = "5.7.8";
 
-  nativeBuildInputs = [ meson ninja pkg-config python3 qttools wrapQtAppsHook ];
-
-  buildInputs = [ radare2 qtbase ];
+  srcs = [
+    (fetchFromGitHub rec {
+      owner = "radareorg";
+      repo = "iaito";
+      rev = version;
+      hash = "sha256-c36WLpVUnffeY6cXSEHvguo8BHyxaLAluN9hBKsQc0s=";
+      name = repo;
+    })
+    (fetchFromGitHub rec {
+      owner = "radareorg";
+      repo = "iaito-translations";
+      rev = "ab923335409fa298c39f0014588d78d926c6f3a2";
+      hash = "sha256-qkIC67a6YRwOa2Sr16Vg6If1TmAiSKUV7hw13Wxwl/w=";
+      name = repo;
+    })
+  ];
+  sourceRoot = "iaito/src";
 
   postUnpack = ''
-    sourceRoot=$sourceRoot/src
+    chmod -R u+w -- iaito-translations
```

```suggestion
    chmod -R u+w iaito-translations
```
FYI we do not need that here